docs/source/api/stats.rst

```diff
@@ -1,25 +0,0 @@
-*****
-Stats
-*****
-Statistics and diagnostics are delegated to the
-`ArviZ <https://arviz-devs.github.io/arviz/index.html>`_.
-library, a general purpose library for
-"exploratory analysis of Bayesian models."
-For statistics and diagnostics, ``pymc3.<function>`` are now aliases
-for ArviZ functions. Thus, the links below will redirect you to
-ArviZ docs:
```

Comment:
I think we should keep these lines around, at least for 3.11, to ease the transition, just like we're doing for `plots.rst`. So, something like that:
```raw
*****
Stats
*****
Statistics and diagnostics are delegated to the
`ArviZ <https://arviz-devs.github.io/arviz/index.html>`_.
library, a general purpose library for
"exploratory analysis of Bayesian models."
Refer to its documentation to use the diagnostics functions directly.
```